opencti-platform/opencti-front/src/private/components/import/ImportContent.jsx

```diff
@@ -171,17 +172,18 @@ const importValidation = (t, configurations) => {
   return Yup.object().shape(shape);
 };
 
+const initState = {
```

Comment:
I don't think we want to initialize that outside of the react render

Reply:
I think if we put inside the react render, we will initialize each time the state. What do you think ?

Reply:
Yes, but I don't think we want a object to be living with some value even when the component as been destroyed. Not really confident about the best practices with react. @guillaumejparis a thougth ?

Reply:
No difference for me, the state will be initialized with this object but after the state will be handle internally no action will be done on this object after and it will be destroyed when the js garbage collector passes.



> I think if we put inside the react render, we will initialize each time the component is render. What do you think ?

No danger here, we are in old component so no re render of the constructor

Reply:
Finally, I get back to the initial configuration. I did that because there were some bugs in closing the import, the selectedConnector wasn't reset and I thought that would be a good idea to reset to the initialState but here we don't necessarily need that.